modules/test/conn/python/src/connection_module.py

```diff
@@ -74,6 +74,60 @@ def __init__(self, module):
     # response = self.dhcp1_client.set_dhcp_range('10.10.10.20','10.10.10.30')
     # print("Set Range: " + str(response))
 
+  def _connection_switch_arp_inspection(self):
+    LOGGER.info('Running connection.switch.arp_inspection')
+
+    no_arp = True
+
+    # Read all the pcap files
+    packets = rdpcap(STARTUP_CAPTURE_FILE) + rdpcap(MONITOR_CAPTURE_FILE)

```

Comment:
Inspecting the startup capture causes false failures because the two DHCP servers will get their own responses from the device with different IP's. 

![image](https://github.com/google/testrun/assets/50999916/06cf34d4-17b7-430a-9e20-63940ac34464)

Attempting to just inspect the monitor appears to show similar issues where the ARP who has packets from the dhcp servers during startup appear to propagate, maybe through arp caching into the monitor and still get responses from each IP address.  This may require a more complex process to detect bad ARP packets with the usage of DHCP failover because this does not appear to be a failure of the device.

Reply:
I'm not sure I'm following, I am not getting any failures when running this test on physical devices even with DHCP failover.

Reply:
I can no longer recreate the failure I was seeing when this was reported.  I can't find any difference between my testing scenario so I'm ok moving forward with this.